### ISSUE TITLE
Workspace: Context menu persists after clicking "Collapse Lane"

### ISSUE THREAD

**Issue by nickgros** [gitbutlerapp/gitbutler#4444]
### Version

0.12.12

### Operating System

Linux

### Distribution Method

deb (Linux)

### Describe the issue

I have a few virtual branches open in my workspace. When I click "..." to open the context menu and then click "Collapse Lane", the branch collapses and the context menu persists on the screen. I cannot interact with anything else on the screen except for the context menu. I must close and re-open the app to recover.

Some system specs that may or may not be relevant:

```
OS: Pop!_OS jammy 22.04 x86_64
Kernel: Linux 6.9.3-76060903-generic
DE: GNOME 42.9
WM: Mutter (X11)
```

### How to reproduce

1. Open workspace
2. Add one or more virtual branches
3. Attempt to collapse a virtual branch

### Expected behavior

The context menu disappears after collapsing the branch.

### Relevant log output

```shell
[Error] TypeError: null is not an object (evaluating 't(ft).show')
	r (user.DtrcbDXW.js:19:5226)
```

**Comment by Byron** [comment]
Thanks for reporting! 

I can reproduce this on MacOS as well.

<img width="397" alt="Screenshot 2024-07-19 at 21 19 43" src="https://github.com/user-attachments/assets/a1552cd2-adcd-40bb-a873-c4cb3eba1abf">

Also CC @ndom91 who might know exactly what to do here.

**Comment by PavelLaptev** [comment]
I can't reproduce this issue on the development version, but it occurs on the production `0.5.615` on macOS.

**Comment by PavelLaptev** [comment]
@ndom91 if you don't mind, I'll assign this to you since I can't reproduce the issue. Maybe you can check it on your end.

**Comment by Byron** [comment]
Actually, I also can't reproduce it on 02d4d6fb63a7c5a9268611ea577f4fac176abbe9, the latest `master` at the time.
Since it reproduces on the latest nightly, the issue was fixed between 2a08b419eaf5f444f910faa1049ac033ca94e47a and 02d4d6fb63a7c5a9268611ea577f4fac176abbe9, so we can probably consider this issue resolved.

For good measure, I also built a nightly locally from 02d4d6fb63a7c5a9268611ea577f4fac176abbe9 and could validate that the issue indeed is gone. 

Closing based on these findings, but please let us know the nightly that should be available tomorrow at the latest still doesn't work for you.
